CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
+* Includes:
+  * This definition of **public API**.
+  * The configuration defaults, schemas, and paths where configuration files are stored.
+  * The Command Line Interface.
+  * The transaction schema.
+  * All the endpoints exposed by the HTTP API (including payloads sent to and received from them).
```

Comment:
Might want to mention the WebSocket Event Stream API here explicitly as well.

We defined it here as well: https://github.com/bigchaindb/bigchaindb/pull/1547/files#diff-57fcbb9dfad73b9008ffed4bbbb799ddR10